engine/graphics/src/vulkan/graphics_vulkan.cpp

```diff
@@ -148,15 +148,6 @@ namespace dmGraphics
         DM_STATIC_ASSERT(sizeof(m_TextureFormatSupport)*4 >= TEXTURE_FORMAT_COUNT, Invalid_Struct_Size );
     }
 
-    Context::~Context()
-    {
-        if (m_Instance != VK_NULL_HANDLE)
-        {
-            vkDestroyInstance(m_Instance, 0);
-            m_Instance = VK_NULL_HANDLE;
-        }
-    }
-
```

Comment:
Got a linker issue with dmGraphics::~Context already defined in the opengl objects, so moved to the VulkanDestroyContext function instead